docs/README.md

```diff
@@ -1,503 +1,491 @@
 
-```html
+```ts
+// from CDN
 <script src="https://checkout.airwallex.com/assets/elements.bundle.min.js"></script>
+
+Airwallex.init({
+  env: 'demo', // Setup which Airwallex env('demo' | 'prod') to integrate with
+  origin: window.location.origin, // Set up your event target to receive the browser events message
+});
 ```
 
-Running `init` and embedding the above script in your document head will initialize a global variable in your document called `window.Airwallex`. This variable consists of all the functions to process payments.
+`init()` and `loadAirwallex()` support various options to set up the payment environment as shown in the following table.
 
-| Option   | Default  | Description                                                                                    |
-| -------- | -------- | ---------------------------------------------------------------------------------------------- |
-| `env`    | `'prod'` | Indicate which airwallex integration env your merchant site would like to connect with         |
-| `origin` | `''`     | Your checkout website origin url, aka merchant checkout page's 'window.location.origin' field. |
-| `locale` | `'en`    | i18n localization config, 'en' or 'zh'                                                         |
-| `fonts`  | []       | Fonts options used to customize the payment elements                                           |
+### `options` object properties
+
+| **Property** | **Required?** | **Type**         | **Default value** | **Description**     |
+| ------------ | ------------- | ---------------- | ----------------- | ------------------- |
+| `env`        | No            | enum             | `'prod'`          | The Airwallex environment you want to integrate your application with.   |
+| `origin`     | No            | string           | `''`              | Your website’s origin url – `window.location.origin` field.      |
+| `locale`     | No            | string           | `'en'`            | The locale for your website. Options include: `en`, `zh`, `ja`, `ko`, `ar`, `fr`   |
+| `fonts`      | No            | FontOptions\[ \] | \[ \]             | Fonts options to customize the font styles of payment Elements. Airwallex supports two font weights: regular (400) and bold (700).  You can specify font options using `src`, `family` and `weight` attributes. |
+
+**Note** The Airwallex package only needs to be initialized once in your application.
 
 <br>
 
-## Interacting with Elements
+## Interact with Embedded Elements
+
+### Create an Element
 
-### Element
+This method creates an instance of an individual Element. It takes the `type` of Element to create and an optional `options` object. Elements are rendered as iframes. 
 
-Functions and field components that can be used during checkout integration. Elements are rendered as iframes.
+```ts
+Airwallex.createElement(type, options)
```

Comment:
if customer is using npm, the code will be the following:
```
import { createElement } from '@airwallex/airwallex-payment-elements';
const element = createElement(type, options)
```

Reply:
Updated.